src/Components/Footer/Footer.scss

```diff
@@ -1,17 +1,14 @@
-@import '../commonStyles/reset.scss';
-@import '../commonStyles/variables.scss';
-@import '../commonStyles/font-family.scss';
+@import './../../commonStyles/reset.scss';
```

Comment:
try to move reset and font-family imports to App.scss in order to have it on top level instead of some random component. And don't forget to import App.scss in App.js file to make it work